`var` interface change and docs (#1962)
foundry-rs/starknet-foundry: docs/src/appendix/snforge-library/env/var.md

# `var`

> `fn var(name: ByteArray) -> Array<felt252>`

Reads an environment variable, without parsing it.

> The serialized output is correlated with the inferred input type, as in the [file reading docs](../fs/read_txt.md#accepted-format).

Reviewer: ```suggestion
The serialized output is correlated with the inferred input type, same as during [reading a file](../fs/read_txt.md#accepted-format).
```
